image_bootstrap/patches/cloud-init-0-7-6-pkgbuild.patch

```diff
@@ -33,6 +33,9 @@
 +    sed -i '1s/python$/&2/' $pkgname-$PKGVER/tools/read*
 +    sed -i 's:/etc/systemd:/usr/lib/systemd:g' $pkgname-$PKGVER/setup.py
 +    sed -i 's,name: ubuntu,name: archlinux,;s,gecos: Ubuntu,gecos: Archlinux,' $srcdir/$pkgname-$PKGVER/config/cloud.cfg
++
++    cd ${srcdir}/${pkgname}-${PKGVER}
++    patch -Np1 -i "${OLDPWD}"/../cloud-init-0-7-6-uid.patch
```

Comment:
Thanks for that. I made it work with `${srcdir}` now, please check the new version below.